functions/pipeline/onboarding/__init__.py

```diff
@@ -9,10 +26,106 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
         req_body = req.get_json()
         logging.error(req.get_json())
         url_list = req_body["imageUrls"]
-        url_string = (", ".join(url_list))
     except ValueError:
         print("Unable to decode JSON body")
         return func.HttpResponse("Unable to decode POST body", status_code=400)
 
     logging.error(req_body)

```

Comment:
Is there a logging.debug function so we don't log the body as an error when deployed to production?